packages/mds-types/index.ts

```diff
@@ -338,9 +338,8 @@ export interface ComplianceResponse {
 // We don't put the publish_date into the geography_json column
 // as we do with the Policy type, because we don't want to mess with
 // the geojson FeatureCollection type.
-export interface Geography {
+interface BaseGeographyType {
```

Comment:
I don't like this implementation, seems like extra overhead. Should just keep the original `Geogrpahy` type and then do:
`export type GeographySummary = Omit<Geography, 'geography_json'>`

Reply:
Ah neat, I forgot about `Omit`.